x-pack/legacy/plugins/ml/public/data_frame/pages/job_management/components/job_list/job_service/stop_job.ts

```diff
@@ -8,26 +8,30 @@ import { i18n } from '@kbn/i18n';
 import { toastNotifications } from 'ui/notify';
 import { ml } from '../../../../../../services/ml_api_service';
 
-import { DataFrameJobListRow } from '../common';
+import { refreshTransformList$, REFRESH_TRANSFORM_LIST_STATE } from '../../../../../common';
 
-import { GetJobs } from './get_jobs';
+import { DATA_FRAME_TASK_STATE, DataFrameJobListRow } from '../common';
 
-export const stopJobFactory = (getJobs: GetJobs) => async (d: DataFrameJobListRow) => {
+export const stopJob = async (d: DataFrameJobListRow) => {
```

Comment:
As this is a new function, is it worth going with `stopTransform` rather than `stopJob`? Or are you planning a big rename later?